packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -82,7 +82,7 @@ export const components: ThemeOptions["components"] = {
       }),
       message: ({ ownerState, theme }) => ({
         padding: 0,
-        lineHeight: theme.typography.body.lineHeight,
+        lineHeight: theme.typography.body1.lineHeight,
```

Comment:
What's `body1`?

Reply:
`body1` is MUI's default for this typography style. We had previously simplified this to `body`; however, MUI utilizes the style in different places, so I've changed the name back for now.